src/common/hostPlatform.ts

```diff
@@ -101,6 +112,10 @@ class LinuxHostPlatform extends UnixHostPlatform {
     public getPlatformId(): HostPlatformId {
         return HostPlatformId.LINUX;
     }
+
+    public getUserID(): string {
+        return process.env.USER;
```

Comment:
as far as I know that's the right environment variable to get the username in Windows/Linux/OSX